docs/components/Footer/index.jsx

```diff
@@ -0,0 +1,16 @@
+import styles from '../../styles/Home.module.css'
```

Comment:
We generally don't want this committed for a tailwind site - i think you want to have booted the app with this example https://github.com/vercel/next.js/tree/canary/examples/with-tailwindcss.

Reply:
Noted.